include/openthread/types.h

```diff
@@ -760,27 +760,30 @@ typedef enum otRoutePreference
     OT_ROUTE_PREFERENCE_HIGH = 1,   ///< High route preference.
 } otRoutePreference;
 
-/**
- * This structure represents a whitelist entry.
- *
- */
-typedef struct otMacWhitelistEntry
+#define OT_MAC_FILTER_ITERATOR_INIT  0     ///< Initializer for otMacFilterEntry.
```

Comment:
nit: `///< Initializer for otMacFilterEntry` -> `///< Initializer for otMacFilterInterator`